src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1175,8 +1195,21 @@ namespace winrt::TerminalApp::implementation
     // Arguments:
     // - sender: the control that originated this event
     // - eventArgs: the event's constituent arguments
-    void TerminalPage::_OnTabItemsChanged(const IInspectable& /*sender*/, const Windows::Foundation::Collections::IVectorChangedEventArgs& /*eventArgs*/)
+    void TerminalPage::_OnTabItemsChanged(const IInspectable& /*sender*/, const Windows::Foundation::Collections::IVectorChangedEventArgs& eventArgs)
```

Comment:
Out of curiosity, is this just called a bunch of times as tabs are dragged? Like (removed, 4), (added, 3), (removed, 3), (added, 2), etc?

Reply:
Yes it is.

Reply:
Fixed in next commit